src/main/java/frc/robot/RobotContainer.java

```diff
@@ -67,8 +67,22 @@ public RobotContainer(RobotId robotId) {
     switch (robotId) {
 
       case TestBed1:
+        // GameController kDriver = new GameController(Constants.DRIVER_JOY);
+        // kDriver.get(Button.X).onTrue(new InstantCommand(() -> kDriver.setRumble(RumbleStatus.RUMBLE_ON)));
+        // kDriver.get(Button.X).onFalse(new InstantCommand(() -> kDriver.setRumble(RumbleStatus.RUMBLE_OFF)));
+        intake = new Intake();
         index = new StorageIndex();
-        shooter = new Shooter();
+        operator = new Operator(intake, index);
+        operator.configureControls((x)->{
```

Comment:
this should not be in robot container. move it to the operator class

Reply:
oh, and also please rename all instances of variables named "x" to something more explanatory, like hasNote